tests/test_models.py

```diff
@@ -51,3 +51,53 @@ def test_roles(self):
         person.roles.append(role)
         assert role in person.roles
         assert person.has_roles('admin')
+
+
+@pytest.mark.usefixtures('db')
+class TestCarpool:
+    """Carpool tests."""
+
+    def test_get_ride_requests_query(self, db):
+        """Test get ride requests query"""
+        carpool_1 = CarpoolFactory()
+        carpool_2 = CarpoolFactory()
+
+        person = PersonFactory()
+
+        ride_request_1 = RideRequest(
```

Comment:
To cut down on the boilerplate of instantiating a `Person`, we could implement this as a `RideRequestFactory` with a `SubFactory` to create the `Person`.

Reply:
done